presto-druid/src/test/java/io/prestosql/plugin/druid/DruidIntegrationSmokeTest.java

```diff
@@ -0,0 +1,149 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.prestosql.plugin.druid;
+
+import io.prestosql.testing.AbstractTestIntegrationSmokeTest;
+import io.prestosql.testing.QueryRunner;
+import org.intellij.lang.annotations.Language;
+import org.testng.annotations.AfterClass;
+import org.testng.annotations.Test;
+
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+
+import static io.prestosql.tpch.TpchTable.ORDERS;
+
+@Test
+public class DruidIntegrationSmokeTest
+        extends AbstractTestIntegrationSmokeTest
+{
+    private TestingDruidServer druidServer;
+
+    public DruidIntegrationSmokeTest()
+    {
+        try {
+            Path storage = Paths.get("storage");
+            if (Files.exists(storage)) {
+                druidServer = new TestingDruidServer(storage.toAbsolutePath().toString());
+            }
+            else {
+                druidServer = new TestingDruidServer(Files.createDirectory(storage).toAbsolutePath().toString());
+            }
+        }
+        catch (IOException e) {
+            throw new RuntimeException(e);
+        }
+    }
+
+    @Override
+    protected QueryRunner createQueryRunner()
+            throws Exception
+    {
+        return DruidQueryRunner.createDruidQueryRunnerTpch(druidServer, ORDERS);
+    }
+
+    @AfterClass(alwaysRun = true)
+    public void destroy()
+    {
+        druidServer.close();
+    }
+
+    @Test
+    @Override
+    public void testShowSchemas()
+    {
+        assertQuery("SHOW SCHEMAS FROM druid", "VALUES 'druid', 'information_schema', 'sys', 'lookup'");
+    }
+
+    @Test
+    @Override
+    public void testDescribeTable()
+    {
+    }
```

Comment:
Document why.
See also https://prestosql.slack.com/archives/CP1MUNEUX/p1588690413495700 how we would like to suppress tests now. This also solves the documentation needs.